dataTransform/ProductsTransformer.js

```diff
@@ -1,4 +1,71 @@
-const csv = require("csv-parser");
-const createCsvStringifier = require("csv-writer").createObjectCsvStringifier; const fs = require('fs');
-const Transform = require("stream").Transform;
-const Products = require('../csv_files/product.csv');
+/* eslint-disable no-param-reassign */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-useless-constructor */
```

Comment:
If you want to disable some eslint rules for specific folders (in this case all the files in dataTransform directory), you can add .eslintignore and put the directory inside of .eslintignore similiar to .gitignore